README.md

```diff
@@ -170,6 +170,8 @@ This takes the `data.json` file, masks the data contained inside it and put the
 * `--mask` Declare a simple masking definition in command line (minified YAML format: `--mask "value={fluxUri: 'pimo://nameFR'}"`, or `--mask "value=[{add: ''},{fluxUri: 'pimo://nameFR'}]"` for multiple masks). For advanced use case (e.g. if caches needed) `masking.yml` file definition will be preferred.
 * `--repeat-until <condition>` This flag will make PIMO keep masking every input until the condition is met. Condition format is using [Template](https://pkg.go.dev/text/template). Last output verifies the condition.
 * `--repeat-while <condition>` This flag will make PIMO keep masking every input while the condition is met. Condition format is using [Template](https://pkg.go.dev/text/template).
+* `--stats <filename | url>` or `-S <filename | url>` This flag either outputs run statistics to the specified file or send them to specified url (has to start with `http` or `https`).
```

Comment:
short version should be reserved for more usefull command like --seed or --secure. Please remove the short option.